servlet-galleon-pack/src/main/resources/feature_groups/security.xml

```diff
@@ -19,5 +19,34 @@
                 </feature>
             </feature>
         </feature>
+        <feature spec="subsystem.security.security-domain">
+            <param name="security-domain" value="jboss-web-policy"/>
+            <param name="cache-type" value="default"/>
+            <feature spec="subsystem.security.security-domain.authorization.classic">
+                <feature spec="subsystem.security.security-domain.authorization.classic.policy-module">
+                    <param name="policy-module" value="Delegating"/>
+                    <param name="code" value="Delegating"/>
+                    <param name="flag" value="required"/>
+                </feature>
+            </feature>
+        </feature>
+        <feature spec="subsystem.security.security-domain">
+            <param name="cache-type" value="default"/>
+            <param name="security-domain" value="jaspitest"/>
+            <feature spec="subsystem.security.security-domain.authentication.jaspi">
```

Comment:
Should this be in the default config?

Reply:
The reasons are the same reasons for including it in the legacy variant, discussed before